R/lint.R

```diff
@@ -245,7 +245,9 @@ lint_dir <- function(path = ".", relative_path = TRUE, ..., exclusions = NULL,
 #'   )
 #' }
 #' @export
-lint_package <- function(path = ".", relative_path = TRUE, ..., exclusions = list("R/RcppExports.R"), parse_settings = TRUE) {
+lint_package <- function(
+  path = ".", relative_path = TRUE, ..., exclusions = list("R/RcppExports.R"), parse_settings = TRUE
+) {
```

Comment:
LMK the preferred way to write long function signatures. I'm partial to

```
foo <- function(
  args...
) {
```

But I often see

```
foo <- function(arg1,
                arg2,
                arg2,
                ...) {
```

Reply:
The rest of the codebase, e.g. `lint_dir` and `lint_package` use the latter style. We should be consistent I think.